plugins/Tour/Engagement/ChallengeBrowseMarketplace.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\Tour\Engagement;
+
+use Piwik\Piwik;
+use Piwik\Url;
+
+class ChallengeBrowseMarketplace extends Challenge
```

Comment:
This task never completes, even if I browse the marketplace page. (I opened the page and opened an individual plugin modal.)

Reply:
👍 probably forgot to commit, will look into it